cmd/create/ocmrole/cmd.go

```diff
@@ -499,3 +508,24 @@ func createPermissionPolicy(reporter *rprtr.Object, awsClient aws.Client, policy
 
 	return nil
 }
+
+func checkAdminRoleExists(reporter *rprtr.Object, awsClient aws.Client, roleName string, isAdmin bool) error {
```

Comment:
see if you can make use of this function also in auto mode. Right now this is a code duplication